src/integrationtest/python/GeoAttribute_tests.py

```diff
@@ -0,0 +1,48 @@
+# Name: Debasish Kanhar
+
+import unittest
+from subprocess import Popen, PIPE
+from JanuaGraphContainer import JanusGraphContainer
+from janusgraph_python.driver.ClientBuilder import JanusGraphClient
+from gremlin_python.structure.graph import Graph
+from janusgraph_python.core.attribute.Geo import Geo
+from janusgraph_python.core.datatypes.GeoShape import GeoShape
+
+
+class TestGeoAttributes(unittest.TestCase):
+    def setUp(self):
+        self.container = JanusGraphContainer()
+
+        docker_ip = Popen(["docker-machine", "ip"], stdout=PIPE).communicate()[0]
+        docker_ip = docker_ip.strip().decode("utf-8")
+
+        self.client = JanusGraphClient()
+        self.client = self.client.connect(host=str(docker_ip), port="8182",
+                                          traversal_source="gods_traversal").get_connection()
+
+        self.container.start()
+        self.g = Graph().traversal().withRemote(self.client)
+
+    def tearDown(self) -> None:
+        self.client.close()
+        self.container.stop()
+
+    def test_geo_contains(self):
+
+        # First we will need to add a property so that GeoContains can be queried.
+        place = GeoShape.Circle(50, 100, 100)
+        self.g.V().has("name", "tartarus").property("place", place).next()
```

Comment:
This doesn't appear to be used for anything. Should this be saved to a variable and asserted? Or should these 2 lines be removed?

Reply:
I actually didn't find a query I could do to `GraphOfGods` graph for `GeoContains`. Is there something I'm missing? 

Hence, I add a `GeoShape` which I then query for testing `GeoContains`. 

Logic here is, in `graph of gods` all `geo shapes` are of `point` type thus making a `geocontains` query invalid based on my understanding. Hence I add a bit `circle` object here, and query it using smaller `circles` in tests.